analysis_driver/report_generation/report_crawlers.py

```diff
@@ -24,6 +24,11 @@
     ELEMENT_GENDER_VALIDATION, ELEMENT_GENDER_HETX, ELEMENT_LANE_PC_OPT_DUP, ELEMENT_GENDER_COVY, ELEMENT_SNPS_TI_TV, \
     ELEMENT_SNPS_HET_HOM
 
+# temporary placeholders, import from constants when egcg core released
+ELEMENT_ADAPTER_TRIM_R1 = 'adaptor_bases_removed_r1'
+ELEMENT_ADAPTER_TRIM_R2 = 'adaptor_bases_removed_r2'
```

Comment:
These constants are available in [EGCG_Core v0.4.4](https://github.com/EdinburghGenomics/EGCG-Core/blob/v0.4.4/egcg_core/constants.py#L29)